src/SuperstateToken.sol

```diff
@@ -13,7 +14,7 @@ import {AllowList} from "src/AllowList.sol";
  * @notice A Pausable ERC7246 token contract that interacts with the AllowList contract to check if transfers are allowed
  * @author Superstate
  */
-abstract contract SuperstateToken is ERC20Upgradeable, IERC7246, PausableUpgradeable {
+abstract contract SuperstateToken is ERC20Upgradeable, IERC7246, PausableUpgradeable, Ownable2StepUpgradeable {
```

Comment:
We should also check storage layout after adding this. I forget where it gets added by default and it may (?) depend on the ordering of everything after the `is`

Reply:
Any news on this?